WalletWasabi.Gui/Behaviors/MnemonicSuggestionBehavior.cs

```diff
@@ -50,11 +52,10 @@ private void HandleAutoUpdate()
 			var text = textBox.Text;
 			var enteredWordList = text.Split(' ', StringSplitOptions.RemoveEmptyEntries);
 			var lastWorld = enteredWordList.LastOrDefault();
```

Comment:
typo lastWord :)

Reply:
yes, you're right. I cannot listen the national inflation forecast report and program at the same time.